testbed/Makefile

```diff
@@ -21,14 +21,7 @@ runtests: test
 
 .PHONY: fmt
 fmt:
-	@FMTOUT=`$(GOFMT) -s -l ./.. 2>&1`; \
-	if [ "$$FMTOUT" ]; then \
-		echo "$(GOFMT) FAILED => gofmt the following files:\n"; \
-		echo "$$FMTOUT\n"; \
-		exit 1; \
-	else \
-	    echo "Fmt finished successfully"; \
-	fi
+	$(GOFMT) -s -l ./..
```

Comment:
This was incorrect since it assumed gofmt does not output anything in success case. It does sometimes. Changed to simply use gofmt exit code, all this code was unnecessary.